src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/StaticInvalidationService.kt

```diff
@@ -24,7 +26,8 @@ class StaticInvalidationService {
      * @param lines the lines to invalidate tests
      * @param cache the cache
      */
-    fun invalidateCacheLines(fileUrl: String, lines: Set<Int>, cache: TestCaseCachingService) {
+    fun invalidateCacheLines(fileUrl: String, lines: Set<Int>, project: Project) {
```

Comment:
You don't need to pass the project - you can simply retrieve that from the constructor of this service since it's a project service.

Reply:
addressed